tests/core/creators/manual/detached.md

```diff
@@ -0,0 +1,16 @@
+@bender-tags: 4.17.0, bug, 4461
+@bender-ui: collapsed
+@bender-ckeditor-plugins: toolbar, wysiwygarea, sourcearea, undo, clipboard, basicstyles, elementspath
+
+1. Click first button to attach first editor target element to DOM.
+2. Click second button to attach second editor target element and create it from callback.
```

Comment:
Let's split this test into two since they are independent. We'll have smaller and easier to understand `.html` files then.